doc/source/python_reference.rst

```diff
@@ -139,7 +139,7 @@ Functions:
    read_bem_solution
    read_bem_surfaces
    read_cov
-   read_dip
+   read_dipoles
```

Comment:
I plan on changing a bit about how dipoles are processed, but I could at least isolate the new reading and writing functions if that would make your life easier, yeah